gameover.html

```diff
@@ -27,8 +27,8 @@ <h2>Enter Player Name: </h2>
                 <button class="ui button" type="button">Save</button>
             </div>
         </section>
-        <div class="ui divider"></div>
-        <section id="score" class="ui warning message">
+        <div class="ui-divider"></div>
+        <section id="score" class="ui-warning-message">
```

Comment:
Line 30-31: you changed the classes for this which made it all one class instead of 3 separate classes which messes up the semantic-ui class